website/api/controllers/view-osquery-table-details.js

```diff
@@ -0,0 +1,58 @@
+module.exports = {
+
+
+  friendlyName: 'View osquery table details',
+
+
+  description: 'Display "Osquery table details" page.',
+
+  inputs: {
+    tableName : {
+      description: 'The slug of the osquery table that this user wants to display',
+      example: 'account_policy_data',
+      type: 'string',
+    }
```

Comment:
This is required, no?

Reply:
It is! I've updated the input to be `required: true`.